zerver/lib/hotspots.py

```diff
@@ -46,7 +46,7 @@ def to_dict(self, delay: float = 0) -> Dict[str, Union[str, float, bool]]:
         ),
     ),
     Hotspot(
-        name="intro_gear",
+        name="intro_personal",
```

Comment:
Renaming this without doing a database migration is problematic -- it'll mean the software loses track of which users have already done this hotspot. I think the best answer is to just not rename the hotspot and include a comment that it's a bit of a misnomer as the thing points to the gear menu.

(I think we'll probably replace the hotspots entirely before it's worth renaming these).

Reply:
Excuse me for the delay, and thank you for making these changes! :)